components/tyk-dashboard/values.yaml

```diff
@@ -139,6 +139,11 @@ global:
     #   name: ""
     #   keyName: ""
 
+  # hashKeys is used to enable/disable key hashing. It should match with the value set in Gateway configuration.
+  # The Dashboard will now operate in a mode that is compatible with key hashing.
+  # It is used to set TYK_DB_HASHKEYS
+  hashKeys: true
```

Comment:
Can you also add a small comment on `dashboard.hashKeys` field that it will be deprecated in favor of this new global one?